include/INA138.hpp

```diff
@@ -0,0 +1,49 @@
+#ifndef LVSS_INA138_HPP
+#define LVSS_INA138_HPP
+
+#include "EVT/dev/LCD.hpp"
+#include "EVT/io/ADC.hpp"
+#include "EVT/io/CANopen.hpp"
+#include "EVT/io/GPIO.hpp"
+#include "EVT/io/SPI.hpp"
+#include "EVT/utils/log.hpp"
+#include "LVSS.hpp"
+
+namespace IO = EVT::core::IO;
+namespace DEV = EVT::core::DEV;
+namespace log = EVT::core::log;
+
+namespace LVSS {
+
+/**
+ * class for LVSS current sensor feature
+ */
+class INA138 {
+public:
+    /**
+      * Constructor for current sensing class
+      */
+    INA138(IO::ADC& adc0);
+
+    /**
+      * Calculates current using the input voltage rail into the Vicor
```

Comment:
Don't make your comments specific to the design of the hardware. This driver is just for an IC that reads current. You can include more detail in the LVSS class.
```suggestion
      * Get the current detected by the INA
```